core/agents/tech_lead.py

```diff
@@ -134,6 +134,7 @@ async def plan_epic(self, epic) -> AgentResponse:
         self.next_state.tasks = self.current_state.tasks + [
             {
                 "id": uuid4().hex,
+                "epic_id": epic["id"],
```

Comment:
we can remove this since we agreed not to save all tasks in the state but only tasks from the current epic